tools/hidpp10-dump-page.c

```diff
@@ -95,7 +94,7 @@ main(int argc, char **argv)
 	path = argv[argc - 1];
 	fd = open(path, O_RDWR);
 	if (fd < 0)
-		error(1, errno, "Failed to open path %s", path);
+		fprintf(stderr, "Failed to open path '%s': %s", path, strerror(errno));
```

Comment:
You also need to add `exit(1);` after every replaced `error` call, as that's what it does - prints a message and exits the program.
So, the code would look like this:
 ```suggestion
		fprintf(stderr, "Failed to open path '%s': %s", path, strerror(errno));
		exit(1);
```
(looks like the indentation is misaligned in the suggestion, I think this is a problem with GitHub)

Reply:
@staticssleever668 Okay, thanks. I'll update the MR

Reply:
@staticssleever668 should it be `exit(3)`. Referring to this https://linux.die.net/man/3/error

Reply:
You are right, good catch. 👍🏿

Reply:
Done :)

Reply:
It actually meant third section of man pages, not literally `exit(3)`, so it should have indeed been `exit(1)`...

Returning `1` is an informal way of saying there is any kind of error.
Not that it matters in any way, so keeping it at `exit(3)` is okay. :)

Reply:
Ahh, shoot. My bad. I need to get better at reading man pages. If you want I can definitely send a second PR to fix this.

Reply:
No, i's good, as I said, it's not like it really matters, so it's okay. :)